src/hackerspace_online/management/commands/full_clean.py

```diff
@@ -0,0 +1,58 @@
+from django.apps import apps
+from django.core.exceptions import ValidationError
+from django.core.management.base import BaseCommand
+from django_tenants.utils import schema_context
+from tenant.models import Tenant
+
+
+class Command(BaseCommand):
+    LOCAL_APPS = (
```

Comment:
Docstring?  What's the purpose of this command?  Why is it here? Remember that documentation is for our future selves and other developers who don't have any context for the code other than what's here.  In a year from now none of us will remember what this is for, and why it exists. So all our unwritten context that we have needs to be added tot he documentation. Link to issues on github, paste in stuff that we've discussed, however you want to provide future developers an understanding of why this exists.  Text is free.

Reply:
done